### PR TITLE
Add `array-callback-return` ESLint rule

### PR DESCRIPTION
**Pull request by ehmicky**
This adds the [`array-callback-return`](https://eslint.org/docs/rules/array-callback-return) ESLint rule. 

This is a useful linting rule, and actually I think applying it in this PR might have found some bugs.

### CODE REVIEW

File: src/functions-templates/js/google-analytics/google-analytics.js [hunk 1]
```diff
@@ -107,8 +109,8 @@ sr: 1440x900
 vp: 945x777
 je: 0
 _u: blabla~
-jid: 
-gjid: 
+jid:
+gjid:
```
Comment (ehmicky) [inline src/functions-templates/js/google-analytics/google-analytics.js:113]: Unrelated change. Not sure why Prettier/ESLint did not pick this.

File: tests/utils/dev-server.js [hunk 2]
```diff
@@ -35,7 +35,7 @@ const startServer = async ({ cwd, env = {}, args = [] }) => {
           close: async () => {
             selfKilled = true
             const pids = await pidtree(ps.pid).catch(() => [])
-            pids.forEach((pid) => () => {
+            pids.forEach((pid) => {
```
Comment (ehmicky) [inline tests/utils/dev-server.js:38]: I believe this was a bug. What do you think @erezrokah ?

**Review by erezrokah — APPROVED** [review]

File: tests/utils/dev-server.js (hunk 2 above)
Comment (erezrokah) [reply tests/utils/dev-server.js:38]: Yes I think so. A bad shameful copy paste probably
